a11y-input/test/ally-input.test.js

```diff
@@ -28,7 +28,8 @@ describe('a11y input', () => {
       <a11y-input></a11y-input>
     `));
 
-    expect(el.shadowRoot.innerHTML).to.equal(`
+    // ref: @open-wc chai plugin https://open-wc.org/testing/semantic-dom-diff.html
+    expect(el).shadowDom.to.equal(`
```

Comment:
#### From https://dev.to/open-wc/testing-workflow-for-web-components-g73#how-does-shadowdomtoequal-work

## How does `expect(el).shadowDom.to.equal()` work?

1. It gets the `innerHTML` of the shadow root.
1. The browser parses the content (no library needed).
1. Normalizes it (every tag/property is placed on its own line).
1. Parses and normalizes the _expected_ HTML string.
1. Passes **_both_** normalized DOM strings to `chai`'s default compare function.
1. In case of a failed comparison, it groups and displays the difference in a clear manner.